druid/metadata.csv

```diff
@@ -0,0 +1,114 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+druid.service.health,gauge,,,,"1 if the service is healthy, 0 otherwise",0,druid,health
+druid.query.time,gauge,,millisecond,,Milliseconds taken to complete a query.,-1,druid,qry time
```

Comment:
whats orientation -1?

Reply:
1 => an increase of the metric is good (for instance the money you make)
-1 => a decrease of the metric is good  (for instance the latency of your service)

Reply:
Not sure to understand your question :)

From the doc about the orientation:
> Set to 1 if the metric should go up, i.e myapp.turnover. Set to 0 if the metric variations are irrelevant. Set to -1 if the metric should go down, i.e myapp.latency.

https://docs.datadoghq.com/developers/integrations/new_check_howto/

Reply:
why should miliseconds taken go down?

Reply:
`druid.query.time` is milliseconds taken to complete a query.

Basically a duration, if it's low, it's good, hence -1.

Reply:
So what are these queries? SQL like in which they are all different (and then it is expected that more complex ones take longer) or are all similar in which case the time going up is an indication of something going wrong?

Reply:
For druid `druid.query.time` is a moving average for a specific duration: https://github.com/apache/incubator-druid/blob/3f3162b85e269749bf0aa848f435409b3a1db8fe/extensions-contrib/moving-average-query/src/main/java/org/apache/druid/query/movingaverage/MovingAverageQueryRunner.java#L141-L143

If the average increases it might mean there is an issue.

Reply:
Few other similar cases:

- https://github.com/DataDog/integrations-core/blame/96a5e692b62f290bf183a8d78f04beb6306ff019/envoy/metadata.csv#L397
- https://github.com/DataDog/integrations-core/blame/e3a4f88aead12d088a7ac372e7d297c8469bd9d0/openldap/metadata.csv#L20
- https://github.com/DataDog/integrations-core/blame/fd71e1b474d4fa749e1f4d785a436a5e2780543c/pgbouncer/metadata.csv#L14